### ISSUE TITLE
关于训练轮数和loss不变问题

### ISSUE THREAD

**Issue by bixueting** [luost26/diffusion-point-cloud#17]
感谢作者的分享，这里想请教几个问题？
（1）我尝试进行训练，用训练后的模型进行测试后结果不好，形状没有成好，是因为训练的不够久吗，需要训练多久呢?
（2）在训练的过程中，我发现loss并没有下降，并且用的是shapenet数据集，这是因为什么呢，形状没有成好，和这是否有关呢？

**Comment by luost26** [comment]
which model are you training? autoencoder or generator?

**Comment by bixueting** [comment]
I trained the gen model~

**Comment by bixueting** [comment]
> I trained the gen model~



>

**Comment by bixueting** [comment]
> which model are you training? autoencoder or generator?

I trained the Generator model.

**Comment by luost26** [comment]
Hi, 

I tested the training script and there seems to be no problem. Can you provide us with more details? My email address is: luost26@gmail.com

**Comment by ZhangXGe** [comment]
Hi, I ran the code directly and encountered a similar problem with the loss curve shown below, can you point out the problem? What should a normal loss curve look like?
![image](https://user-images.githubusercontent.com/44660600/173507432-8b57561a-3db6-498b-8da0-4db6f113526f.png)
